src/components/filterModal/statsList/StatsList.vue

```diff
@@ -90,6 +92,52 @@ export default {
                 }
             });
         },
+        hasFiltered(data) {
+            if (this.searchQuery) {
+                let array;
+                if (Array.isArray(data.data)) {
+                    array = data.data;
+                } else if (data.data.variants) {
+                    // INHERITANCE - custom handler
+                    array = data.data.variants.concat(data.data.family);
+                }
+                const result = array.filter(item => (Array.isArray(item) ? item[0].toLowerCase()
+                    .includes(this.searchQuery.toLowerCase()) : false));
+                return !!result.length;
+            }
+            return false;
+        },
+        hasSecondaryFiltered(data) {
+            return !!data.filter(item => (item.name ? item.name.toLowerCase()
+                .includes(this.searchQuery.toLowerCase()) : false)).length;
+        },
+        hasPrimaryFiltered(data) {
+            if (data.data && Array.isArray(data.data)) {
+                const filter = data.data.filter(item => (Array.isArray(item) ?
+                    this.hasFiltered(data) : this.hasFiltered(item)));
+                return filter.length ? true : this.hasSecondaryFiltered(data.data);
+            }
+            return false;
+        },
+        filteredData(stat) {
+            let result = stat.data;
+            if (stat.type !== 'zygosity' && Array.isArray(result)) {
```

Comment:
I guess you can remove this one from condition: stat.type !== 'zygosity', if you swap them. 
Also Array.isArray relates to enum and status types, is it right?